locale/en-US/add-on.properties

```diff
@@ -109,6 +109,7 @@ incoming_call_title2=Conversation Request
 incoming_call_block_button=Block
 hangup_button_title=Hang up
 hangup_button_caption2=Exit
+hangup_message=You are no longer sharing your tabs
```

Comment:
Where is this string supposed to be used? The name of the string `hangup_message` doesn't seem to match the string.